tests/Dockerfile

```diff
@@ -51,8 +51,8 @@ RUN dnf -y builddep /tmp/cockpit.spec && dnf clean all
 RUN mkdir -p /usr/local/bin /home/user /build /secrets /cache/images /cache/github
 ADD cockpit-tests install-service /usr/local/bin/
 
-RUN usermod -a -G mock user
-RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+RUN usermod -a -G mock user && useradd -a -G wheel admin
+RUN printf 'user ALL=(ALL) NOPASSWD: ALL\nadmin ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
```

Comment:
I suppose it's not relevant, but this now lacks a terminating newline in the `admin` line. It was there before as echo automatically terminates. But fine for me if you tested this.